code/modules/projectiles/guns/projectile/sniper.dm

```diff
@@ -188,3 +188,7 @@
 		breakthings = TRUE
 	else if(damage > 25)
 		icon_state = "gauss"
+
+/obj/item/projectile/bullet/sniper/accelerator/laser
```

Comment:
So this "laser" is actually just a bullet that comes from a laser cannon? Did you test this to ensure that it:
- Deals burn damage, like a laser does?
- Is blocked or has damage reduced according to laser armor, and not bullet?
- Has the correct sprite and sound?